src/components/chips/js/chipsController.js

```diff
@@ -170,18 +177,40 @@ MdChipsCtrl.prototype.init = function() {
   var ctrl = this;
 
   // Set the wrapper ID
-  ctrl.wrapperId = '_md-chips-wrapper-' + ctrl.$mdUtil.nextUid();
+  this.wrapperId = '_md-chips-wrapper-' + this.$mdUtil.nextUid();
 
   // Setup a watcher which manages the role and aria-owns attributes
-  ctrl.$scope.$watchCollection('$mdChipsCtrl.items', function() {
-    // Make sure our input and wrapper have the correct ARIA attributes
-    ctrl.setupInputAria();
-    ctrl.setupWrapperAria();
-  });
+  this.deRegister.push(
+    this.$scope.$watchCollection('$mdChipsCtrl.items', function() {
+      // Make sure our input and wrapper have the correct ARIA attributes
+      ctrl.setupInputAria();
+      ctrl.setupWrapperAria();
+    })
+  );
+
+  this.deRegister.push(
+    this.$scope.$watch('$mdChipsCtrl.items', function() {
```

Comment:
The comment here is the same as above. What's the reason that this new `$watch` is needed?

Reply:
You are just pointing out the replacement of `ctrl` with `this`, right? Then see comment above^^

Reply:
Nope, I'm asking about the two comments "Make sure our input and wrapper have the correct ARIA attributes" and the existing `$watchCollection` versus the new `$watch`.

Reply:
Just added the unwatcher-handling and replaced `ctrl` with `this`
![screenshot_20180226_101737](https://user-images.githubusercontent.com/1506919/36662386-8b9466e6-1ade-11e8-96b3-bf2d89723b56.png)

Reply:
Wait... this is duplicated. Fix it asap